common/views/components/ModalMoreFilters/ModalMoreFilters.tsx

```diff
@@ -112,6 +114,16 @@ const FiltersHeader = styled(Space).attrs({
   width: 100%;
 `;
 
+const OverrideLabelContainer = styled(BaseLabelContainer)`
```

Comment:
Why do we need this override? Could we leave a note so we remember the reason when we come back to this in the future.

Reply:
Agreed, not the right solution for this and as discussed implement a universal solution.